src/main/java/cl/transbank/onepay/example/controller/RefundController.java

```diff
@@ -15,16 +14,9 @@
     @RequestMapping(value = "/refund", method = RequestMethod.GET)
     public ModelAndView refundCreate(@RequestParam("amount") long amount,
                                      @RequestParam("occ") String occ,
-                                     @RequestParam("externalUniqueNumber") String externalUniqueNumber,
-                                     @RequestParam("authorizationCode") String authorizationCode) {
-        Onepay.setIntegrationType(Onepay.IntegrationType.TEST);
-
-        // create options to send Onepay's keys
-        Options options = Options.getDefaults()
-                .setApiKey(ComerceConfig.ONEPAY_API_KEY)
-                .setSharedSecret(ComerceConfig.ONEPAY_SHARED_SECRET);
+                                     @RequestParam("externalUniqueNumber") String externalUniqueNumber,                                     @RequestParam("authorizationCode") String authorizationCode) {
         try {
-            Refund.create(amount, occ, externalUniqueNumber, authorizationCode, options);
+            Refund.create(amount, occ, externalUniqueNumber, authorizationCode);
```

Comment:
How you obtain the authorizationCode now? It was removed from request.

Reply:
No, it wasn't. But yeah, I messed the format and left it at the end of the previous line. I'll fix the format.